worlds/faxanadu/Options.py

```diff
@@ -0,0 +1,106 @@
+from Options import PerGameCommonOptions, Toggle, DefaultOnToggle, StartInventoryPool, Choice
+from dataclasses import dataclass
+
+
+class KeepShopRedPotions(Toggle):
+    """
+    Prevents the Shop's Red Potion to be shuffled. Those locations
+    will have purchasable Red Potion as usual for their usual price.
+    """
+    display_name = "Keep Shop Red Potions"
+
+
+class IncludePendant(Toggle):
+    """
+    Pendant is an item that boosts your attack power permanently when picked up.
+    However, due to a programming error in the original game, it has the reserve
+    effect. You start with the Pendant power, and lose it when picking
+    it up. So this item is essentially a trap.
+    There is a setting in the client to reverse the effect back to its original intend.
+    This could be use in conjunction with this option to increase or lower difficulty.
+    """
+    display_name = "Include Pendant"
+
+
+class IncludePoisons(DefaultOnToggle):
+    """
+    Wether or not to include Poison Potions in the pool of items. Including them
+    effectively turn them into traps in multiplayer.
+    """
+    display_name = "Include Poisons"
+
+
+class RequireDragonSlayer(Toggle):
+    """
+    Requires the Dragon Slayer to be equiped to kill the final boss.
+    Turning this on will turn Progressive Shields into progression items.
+    This setting doesn't make the game forcing you to use the Dragon Slayer
+    to kill the final boss. But it forces rules during generation to make sure
+    you can equip all progressive equipment before reaching that point.
```

Comment:
"equiped" should be "equipped" 
also, a more concise version of this description could be:
"Requires the Dragon Slayer to be available before fighting the final boss is required."

But if you want to keep the current clarity, i'd suggest changing "This setting doesn't make the game forcing you to use the Dragon Slayer to kill the final boss. But it forces rules during generation to make sure you can equip all progressive equipment before reaching that point."
to "This setting does not force you to use Dragon Slayer to kill the final boss. Instead, it ensures that you will have the Dragon Slayer and be able to equip it before you are expected to beat the final boss."

Reply:
I see you took the first part of my suggestion and not the second, 
if you want to keep the 3-5th lines of the description and don't like my alternate wording, this phrase in particular does not read well
"doesn't make the game forcing you" and i still suggest changing that even if you don't take my previous suggestion